src/types/index.ts

```diff
@@ -28,7 +28,9 @@ export interface ArchaeologistProfile {
   minimumDiggingFee: BigNumber;
   maximumRewrapInterval: number;
   peerId: PeerId | string;
-
+  successes: string[];
+  cancels: string[];
```

Comment:
@knoll3 -- are these "cleans", or failures? 

Wondering if these should be a count (rather than array of strings), as the contracts should be updated to return lengths (instead of sarco IDs)